aggregator/src/aggregator.rs

```diff
@@ -117,6 +117,16 @@ pub mod report_writer;
 #[cfg(test)]
 mod taskprov_tests;
 
+pub(crate) fn aggregation_success_counter(meter: &Meter) -> Counter<u64> {
```

Comment:
separate nit/suggestion: would be nice to align the name of this item and of the metric with `aggregate_step_failure_counter`. That said, the other metric's name is bad and needs work (`janus_step_failure` doesn't tell me what was stepped), and we don't want to fix that here since changing metric names involves gymnastics in alerts and Grafana charts.

So concretely: how about `janus_report_aggregation_success_counter`, unless we feel that the `report` unit makes this clear enough?

Reply:
I'm fine with `janus_report_aggregation_success_counter`; I avoided matching the name of the other metric too closely as this tracks success of entire aggregations rather than individual aggregation steps.